tests/Hangfire.PostgreSql.Tests/PostgreSqlStorageFacts.cs

```diff
@@ -40,7 +40,8 @@ public void Connection_Timezone_Is_Set_To_UTC_For_Npgsql6_Compatibility_When_Enl
       using (var connection = storage.CreateAndOpenConnection())
       {
         // Assert
-        Assert.Equal("UTC", connection.Timezone);
+        // test database has GMT+13 time zone set
+        Assert.Equal("'GMT+13'", connection.Timezone);
```

Comment:
<img width="1002" alt="Screen Shot 2023-03-09 at 12 51 46 AM" src="https://user-images.githubusercontent.com/118866349/223943435-6b6b605c-be1f-4d8f-b574-5c0475acd886.png">

Shouldn't this be "UTC+13"? The build server appears to set it as such.

Reply:
Oh, seems like I configured my local env using GMT+13. Changed it now to UTC+13.
However there might be a small catch.. 
If in docker I set 

```
      - TZ='UTC+13'
      - PGTZ='UTC+13'
```

It is also reported with quotes in connection.Timezone field.
If its set to
```
      - TZ=UTC+13
      - PGTZ=UTC+13
```

Then without quotes. I see in appveyor.yml it set directly appending config file like:

```
Add-Content "c:\program files\postgresql\9.6\data\postgresql.conf" "timezone = 'UTC+13'"
```

So not sure which way is reported in build.. cannot see in build history "SHOW TIME ZONE;" output.

Reply:
No quotes in the output:

![image](https://user-images.githubusercontent.com/12134941/223952420-22a54c81-ccea-4a45-bd6a-b9dc4a0e69c2.png)